code/game/objects/items/handcuffs.dm

```diff
@@ -74,10 +74,14 @@
 			else
 				to_chat(user, "<span class='warning'>You fail to handcuff [C]!</span>")
 		else
-			to_chat(user, "<span class='warning'>[C] doesn't have two hands...</span>")
+			to_chat(user, "<span class='warning'>[C == user ? "You don't" : "[C] doesn't"] have two hands...</span>")
 
 /obj/item/restraints/handcuffs/proc/apply_cuffs(mob/living/carbon/target, mob/user, dispense = 0)
-	if(target.handcuffed)
+	if(target.handcuffed) //some sanity checks, in case apply_cuffs() gets called without attack() being called/fully run through (looking at YOU, clumsy people)

```

Comment:
```suggestion
	if(target.handcuffed)
```
Comment no longer relevant, correct?

Reply:
nah, target.handcuffed is still a sanity check, isn't it?

Reply:
What do clumsy people have to do with it?

Reply:
They bypass the target.handcuffed check in attack(), so it needs to be re-checked here.